### DISCUSSION TITLE
Parent context not passing to GRPC

### DISCUSSION THREAD

**Discussion by fir1** [getsentry/sentry-go#368] [Q&A]
There are 2 microservices, each of them starts its own "Span", however, the context of the parent "span" is not passed to the child both microservices are connected through gRPC.

I think the struct type which is linked to the context, should be converted to "metadata" in gRPC, then actual metadata will be attached to the context. Could you please confirm or perhaps you have some examples where you got it working with gRPC. 
Thank you for your support.

**Comment by rhcarvalho** [comment]
Hi @fir1, you need to propagate the `sentry-trace` header. Use [`Span.ToSentryTrace()`](https://pkg.go.dev/github.com/getsentry/sentry-go#Span.ToSentryTrace) and then you can use that value to continue the trace in another service.

Here's an example showing how to continue traces https://gist.github.com/rhcarvalho/3dc943b4340851bc9d590b34661ebf03

**Reply by fir1** [reply]
Hi @rhcarvalho, thank you I can confirm that it is working.
